### PR TITLE
Issue/4406

### PR DESCRIPTION
**Pull request by unho**

### CODE REVIEW

File: docs/server/installation.rst [hunk 1]
```diff
@@ -214,7 +214,7 @@ run it, just issue:
 
 .. code-block:: bash
 
-   (env) $ pootle runserver
+   (env) $ pootle runserver --insecure
```
Comment (dwaynebailey) [inline docs/server/installation.rst]: Needs a big red warning about running with ``--insecure`` just as is in the Django docs.  Or at least point to Django's concern.

Comment (unho) [inline docs/server/installation.rst]: Added a warning with a reference to Django docs.

Comment (dwaynebailey) [inline docs/server/installation.rst]: "Never use it __in__ production."

Otherwise GTM

**Comment by phlax** [comment]
lgtm - +1 to nit

**Pull request merged**